src/sectorfile/RunwayCollection.cpp

```diff
@@ -0,0 +1,172 @@
+#include "pch/stdafx.h"
+#include "sectorfile/RunwayCollection.h"
+#include "euroscope/EuroscopeSectorFileElementInterface.h"
+
+using UKControllerPlugin::SectorFile::Runway;
+using UKControllerPlugin::Euroscope::UserSetting;
+using UKControllerPlugin::Euroscope::EuroscopeSectorFileElementInterface;
+using UKControllerPlugin::SectorFile::SectorFileProviderInterface;
+
+namespace UKControllerPlugin {
+    namespace SectorFile {
+
+
+
+        RunwayCollection::RunwayCollection(SectorFileProviderInterface & sectorFile)

```

Comment:
Could this just be a default constructor `=default` in the header and calling the parent ? Or probably a `= {}`?

Reply:
I dont think so?

Reply:
I dont think so?